### PR TITLE
WIP: Group preliquidity orders as single toast

### LINKED ISSUE #3640: alerts for grouped pre liquidity
UPDATE: Disregard the description below. I talked to Paul about this, and we decided to just show all order created alerts as individual alerts with no grouping.

Create a market with orders. User is directed to the portfolio page. In the My created markets section, there's a button to bring up pre-liquidity modal. 

The orders are batch sent for creation. 0x mesh they all will be sent. We are assuming the same log will come in for 0x order creation.

`Toast can say orders have been placed as this is done a group transaction`
`Bell message should show each individual order`

spreadsheet doesn't differentiate pre-liquidity `Order Placed` creation from regular place order alert. toast message is different. 

Toast Message:
```
Order Placed
[Market Title]
Your orders have been added
```

see excel sheet
https://docs.google.com/spreadsheets/d/1Q18_TKEg9lcSrZX8eWcKt26IjTAc-KNMfBn8oaKBrYc/edit?ts=5d7969a8#gid=0v

### PR DESCRIPTION
**Pull request by adrake33**
Closes #3640

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🥳

 This link will always point to this PR's latest changes: https://peek.run/4kgh5r7

File: packages/augur-ui/src/modules/events/actions/log-handlers.ts [hunk 1]
```diff
@@ -231,7 +231,9 @@ export const handleOrderCreatedLog = (log: Logs.ParsedOrderEventLog) => (
     log.orderCreator,
     getState().loginAccount.address
   );
-  if (isUserDataUpdate) {
+  // Ignore alerts for pre-liquidity orders here.
+  // They are handled elsewhere so they can be grouped as a single alert.
+  if (isUserDataUpdate && log.tradeGroupId === NULL_ADDRESS) {
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/events/actions/log-handlers.ts]: tradeGroupId isn't null for normal orders

Comment (phoebemirman) [reply packages/augur-ui/src/modules/events/actions/log-handlers.ts]: making to so the normal orders aren't being updated with this info and their toasts don't show up

**Comment by phoebemirman** [comment]
are we not showing each order in the bell? not seeing that happen
"Bell message should show each individual order"

**Comment by adrake33** [comment]
Whoops, my mistake. Yes, we should be showing the individual orders under the bell. Will fix.

File: packages/augur-ui/src/modules/alerts/actions/set-alert-text.ts [hunk 2]
```diff
@@ -300,6 +302,25 @@ export default function setAlertText(alert: any, callback: Function) {
...
+        } else {
+          alert.title = 'Orders placed';
+          alert.details = 'Your orders have been added';
+        }
+        dispatch(
+          loadMarketsInfoIfNotLoaded([marketId], () => {
+            const marketInfo = selectMarket(marketId);
+            if (marketInfo === null) return;
+            alert.description = marketInfo.description;
+            alert.toast = true;
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/alerts/actions/set-alert-text.ts]: i think we need to put the toast: true part when the alert first gets added. or else on refresh the toast will show up again

Comment (adrake33) [reply packages/augur-ui/src/modules/alerts/actions/set-alert-text.ts]: Yeah, true. I'll update that as well.

**Review by pgebheim — CHANGES_REQUESTED** [review]
Integrate feedback from chwy :)

File: packages/augur-ui/src/modules/events/actions/log-handlers.ts [hunk 3]
```diff
@@ -21,7 +21,7 @@ import { Action } from 'redux';
 import { AppState } from 'store';
 import { updateBlockchain } from 'modules/app/actions/update-blockchain';
 import { isSameAddress } from 'utils/isSameAddress';
-import { Events, Logs, TXEventName } from '@augurproject/sdk';
+import { Events, Logs, TXEventName, NULL_ADDRESS } from '@augurproject/sdk';
```
Comment (bthaile) [inline packages/augur-ui/src/modules/events/actions/log-handlers.ts]: is NULL_ADDRESS used anywhere?

Comment (adrake33) [reply packages/augur-ui/src/modules/events/actions/log-handlers.ts]: It was used in my initial PR changes. Good catch, will remove.

**Pull request closed without merging**